scripts/prepare_disk.sh

```diff
@@ -62,6 +62,7 @@ partition_resize()
     fi
 
     sfdisk --quiet "${TARGET_DISK}" < "${PARTITION_TABLE_FILE}"
+    sleep 1
```

Comment:
I really do not like this because the test script will take 20+ seconds longer now, although i understand it. 
Maybe 'if there is time' we could just stay in a loop if the storage device is out of sync.